lib/peep.rb

```diff
@@ -0,0 +1,53 @@
+require 'pg'
+require_relative './database_connection.rb'
+
+class Peep
+
+  attr_reader :id, :text, :time, :date, :user_id
+
+  def self.create(text:, user_id:)
+    Peep.set_environment
+
+    @connection = DatabaseConnection.connection
+    clock = Time.new
+    time = "#{clock.hour}:#{clock.min}:#{clock.sec}"
+    date = "#{clock.year}-#{clock.month}-#{clock.day}"
+
+    result = DatabaseConnection.query("INSERT INTO peeps (text, time, date, user_id) VALUES('#{text}', '#{time}', '#{date}', '#{user_id}')RETURNING id, text, time, date, user_id;")
```

Comment:
I think this is where the error about logged_in being nul is happening. Not sure where its being set, maybe you could consider setting default value to true/logged_in etc, when a user signs up?